docs/source/tutorials/quickstart.rst

```diff
@@ -177,19 +182,140 @@ The output should look as follows.
 
-This example shows one of the most basic features of manim: the ability to
+This example shows one of the primary features of manim: the ability to
 implement complicated and mathematically intensive animations (such as cleanly
-interpolating between two geometric shapes) in very few lines of code.
+interpolating between two geometric shapes) with just a few lines of code.
+
+
+Positioning manim objects
+*************************************************
+
+Next, an introduction on how to position manim objects, or mobjects.
+
+1. Open ``scene.py``, and add the following code snippet below the ``SquareToCircle`` method:
+
+.. code-block:: python
+
+   class SquareAndCircle(Scene):
+       def construct(self):
+           circle = Circle()  # create a circle
+           circle.set_fill(PINK, opacity=0.5)  # set the color and transparency
+
+           square = Square()  # create a square
+           square.set_fill(BLUE, opacity=0.5)  # set the color and transparency
+
+           square.next_to(circle, RIGHT, buff=0.5)  # set the position
+           self.play(Create(circle), Create(square))  # show the shapes on screen
+
+2. Render ``SquareAndCircle`` using the following command:
+
+.. code-block:: bash
+
+   manim -pql scene.py SquareAndCircle
+
+The following animation displays:
+
+.. manim:: SquareAndCircle2
+   :hide_source:
+
+   class SquareAndCircle2(Scene):
+       def construct(self):
+           circle = Circle()  # create a circle
+           circle.set_fill(PINK, opacity=0.5)  # set the color and transparency
+
+           square = Square() # create a square
+           square.set_fill(BLUE, opacity=0.5) #set the color and transparency
+
+           square.next_to(circle, RIGHT, buff=0.5) # set the position
+           self.play(Create(circle), Create(square))  # show the shapes on screen
+
+``next_to`` is a mobject method for positioning mobjects. We first specified
+the pink circle as the square's reference point by passing ``circle`` as the method's first argument.
+The second argument is used to specify the direction the Mobject is placed relative to the reference point.
+In this case, we've set the direction to ``RIGHT``, telling manim to position the square to the right of the circle.
+Finally, ``buff=0.5`` applied a small distance buffer between the two objects.
+
+Try changing ``RIGHT`` to ``LEFT``, ``UP``, or ``DOWN`` instead, and see how that changes the position of the square.
+
+Using positioning methods, you can render a scene with multiple mobjects,
+setting their locations in the scene using coordinates or positioning them
+relative to each other.
+
+For more information on ``next_to`` and other positioning methods, check out the
+list of :class:`.Mobject` methods in our reference manual.
+
+
+Using ``.animate`` syntax to animate methods
+*************************************************
+
+The final lesson in this tutorial is using ``.animate``, a mobject method which
```

Comment:
```suggestion
The final lesson in this tutorial is using ``.animate``, a method which
```

Reply:
Is ``.animate`` not a ``Mobject`` method? It seems to modify methods that apply to ``Mobject``s but I don't know for certain what's going on. 

Regardless, I don't think the comma is correct.

Reply:
Yea, I think your wording is fine.